.github/workflows/tests.yml

```diff
@@ -13,22 +13,52 @@ jobs:
             goarch: amd64
```

Comment:
Unrelated to this line, would you consider a log on GCT start to say which one is enabled? Yes, you have build tags, but then it puts it in the logs of the application

Could add to the build tag files to call like `json.Version()` where its `fmt.Println("hello my name is Sonic and I gotta go fast, okay?")` as an example

Reply:
[needs checking](https://github.com/thrasher-corp/gocryptotrader/pull/1623/commits/1c9f9c2ee6adcd8c0c79932526602fe189b4979e)